test/JDBC/expected/BABEL-3486-vu-prepare.out

```diff
@@ -0,0 +1,161 @@
+-- View Conversions with TRY_CONVERT
+CREATE VIEW BABEL_3486_vu_prepare_v1 as (SELECT TRY_CONVERT(varchar(30), CAST('13:01:59' AS time), 8));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v2 as (SELECT TRY_CONVERT(date, '2017-08-25'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v3 as (SELECT TRY_CONVERT(time, '13:01:59'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v4 as (SELECT TRY_CONVERT(datetime, '2017-08-25 13:01:59'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v5 as (SELECT TRY_CONVERT(float, 11234561231231.234));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v6 as (SELECT TRY_CONVERT(MONEY, 55.35));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v7 as (SELECT TRY_CONVERT(float, 1));
+GO
+
+-- Wrong View conversions that return NULL
+CREATE VIEW BABEL_3486_vu_prepare_v8 as (SELECT TRY_CONVERT(date, 123));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v9 as (SELECT TRY_CONVERT(time, 123));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v10 as (SELECT TRY_CONVERT(datetime, 123));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v11 as (SELECT TRY_CONVERT(money, 'asdf'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v12 as (SELECT TRY_CONVERT(float, 'test'));
+GO
+
+-- Procedure conversions with TRY_CONVERT
+CREATE PROCEDURE BABEL_3486_vu_prepare_p1 as (SELECT TRY_CONVERT(varchar(30), CAST('13:01:59' AS time), 8));
+GO
+
+CREATE PROCEDURE BABEL_3486_vu_prepare_p2 as (SELECT TRY_CONVERT(date, '2017-08-25'));
+GO
+
+CREATE PROCEDURE BABEL_3486_vu_prepare_p3 as (SELECT TRY_CONVERT(time, '13:01:59'));
+GO
+
+CREATE PROCEDURE BABEL_3486_vu_prepare_p4 as (SELECT TRY_CONVERT(datetime, '2017-08-25 13:01:59'));
+GO
+
+CREATE PROCEDURE BABEL_3486_vu_prepare_p5 as (SELECT TRY_CONVERT(float, 11234561231231.234));
+GO
+
+CREATE PROCEDURE BABEL_3486_vu_prepare_p6 as (SELECT TRY_CONVERT(MONEY, 55.35));
```

Comment:
What happens if we have >4 digits after the decimal point? Something like 
`SELECT TRY_CONVERT(MONEY, 55.35791)`.

Let's add a test for that.